67 mat-select has wrong label and underline color issue is fixed

raghavenc5 opened the pull request:
<!-- If this pull request fixes an Issue, link it below. If not, you can remove the line below -->

Fixes #67  .

<!-- Include a bulleted list summarizing the main changes you have made in this PR -->

#### Changes proposed in this Pull Request:

-  updated the accent and warn palette to fixed this issue
-
-

<!-- Include screenshots if they will help illustrate the changes in this PR -->

#### Screenshots / Screen Recording (if applicable)

- 
![image](https://user-images.githubusercontent.com/13012853/225909103-3882911f-096d-4a6b-aac3-dcca6e884591.png)


<!-- Instruction for PR reviewers, if more complicated than a simple yarn start -->

#### To Test:

- yarn test

<!-- Useful for draft pull requests -->

#### Any specific feedback you are looking for?

-

huayunh reviewed (commented):
I don't know enough about Angular to do a comprehensive code review... may need to summon @daileytj

raghavenc5 commented on CHANGELOG.md:
```diff
@@ -5,6 +5,7 @@
 ### Fixed
 
 -   Fixed `<blui-drawer-header>` `color` prop issue ([#152](https://github.com/etn-ccis/blui-angular-themes/issues/152)).
+-   Fixed `<mat-select>` wrong label and underline color ([#67](https://github.com/etn-ccis/blui-angular-themes/issues/67P)).
```
Resolved the link issue

raghavenc5 commented:
> 

1. light theme and dark theme, outline select, warn variant: whether something is selected or not, the outline should be red. Right now it is unchanged. -FIXED

<img width="832" alt="Screenshot 2023-03-23 at 1 34 48 PM" src="https://user-images.githubusercontent.com/13012853/227141703-85f9ccdf-2c67-4dd9-92e9-8e61b17625ec.png">

2. dark theme, fill select, warn and required variant: the background color is incorrect. - FIXED

<img width="832" alt="Screenshot 2023-03-23 at 1 35 06 PM" src="https://user-images.githubusercontent.com/13012853/227141883-0141239d-1b3d-464a-923f-079913f7b153.png">

huayunh reviewed (commented):
I just noticed that in dark theme, the background color for mat select filled disabled looks the same as other text fields, but it should be 50% transparent. 

Here's what I'd expect for background colors:

![image](https://user-images.githubusercontent.com/8997218/227214211-9705161b-5424-484b-a5aa-a281092363ca.png)



Everything else looks good. I didn't check the code (not an Angular person)

raghavenc5 commented:
@huayunh The background color for mat select text fields is changed to the correct color format as per your mention. The disabled text field is 50% transparent. see the image below

@daileytj @JeffGreiner-eaton Kindly review the angular code

<img width="362" alt="Screenshot 2023-03-24 at 12 41 55 PM" src="https://user-images.githubusercontent.com/13012853/227469349-95b97b68-210e-48c2-9d8c-6a6c6ed05cc5.png">

huayunh commented:
screenshot looks good
